projects/common/lib/components/phone-number/phone-number.component.html

```diff
@@ -16,7 +16,7 @@
 
 <!-- Error messages for component -->
 <common-error-container
-    [displayError]="!disabled && controlDir?.touched">
+    [displayError]="!disabled && controlDir?.touched && (controlDir?.errors?.required || controlDir?.errors?.incompleteValue)">
```

Comment:
[displayError]="!disabled && (controlDir?.touched || controlDir?.dirty) && controlDir.errors"

Remove required  || incompleteValue for if in common-error-container